src/GeositeFramework/js/Map.js

```diff
@@ -101,20 +100,24 @@ require(['use!Geosite',
         view.model.on('change:extent', function () {
             var currentExtent = view.model.get('extent');
 
-            if (!_.isEqual(currentExtent, view.esriMap.extent)) {
+            if (!_.isEqual(currentExtent, view.esriMapView.extent)) {
                 loadExtent(view);
             }
         });
 
+        /*
         // Configure the esri proxy, for (at least) 2 cases:
         // 1) For WMS "GetCapabilities" requests
         // 2) When it needs to make an HTTP GET with a URL longer than 2000 chars
         esri.config.defaults.io.proxyUrl = "proxy.ashx";
```

Comment:
I don' think so, actually.  A `portal` is a specific esri technology for self hosted clould like environment.  The proxy setting here is really just to point to a server executed redirect.